src/app/showcase/doc/chips/maxvaluesdoc.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, Input } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Code } from '../../domain/code';
+
+@Component({
+    selector: 'max-values-doc',
+    template: ` <section class="py-4">
+        <app-docsectiontext [title]="title" [id]="id">
+            <p>Chips can have a maximum number of entered items. In order to set the limit <i>max</i> property is used and accepts a numeric value which reflects the maximum number of items in the chip list</p>
```

Comment:
I would like to do a suggestion here:
`Chips can have a maximum number of entered items. To set this limit, the <i>max</i> property is used, which accepts a numeric value that represents the maximum number of items in the chip list`

Reply:
Sounds good, will do the change